www/front_src/src/Resources/Listing/api/endpoint.ts

```diff
@@ -3,11 +3,11 @@ import { buildListingEndpoint, ListingParameters } from '@centreon/ui';
 import { resourcesEndpoint } from '../../api/endpoint';
 
 export type ListResourcesProps = {
-  hostGroupIds: Array<number>;
-  monitoringServerIds: Array<number>;
+  hostGroups: Array<number>;
+  monitoringServers: Array<number>;
   onlyWithPerformanceData?: boolean;
   resourceTypes: Array<string>;
-  serviceGroupIds: Array<number>;
+  serviceGroups: Array<number>;
```

Comment:
Are they not `Array<string>` because we filter them by name

Reply:
👍  yep array of strings should be sent